### PR TITLE
BoS feedback tweaks

### PR DESCRIPTION
**Pull request by Leudoberct1**
## About The Pull Request

Adds a bunch of feedback changes to the BoS bunker. Shrinks the space used between the head knight office and firing range, fixes some issues with water tiles in the hydro bay, adds warning signs to the elevator, adds the payroll safe back in, fixes a vending machine, moves the bunker entrance to be further north in the sewers, rather than very close to the Den. Adds some dense rock (still passable) to try and encourage BoS not to mine all the way to their own entrance.

## Why It's Good For The Game

General QoL changes to make it better to play in the bunker, moving the entrance because people didn't like being so close to the Den due to frequent IC hostilities, generally just makes it more enjoyable to play in.

## Pre-Merge Checklist
- [X] Your Pull Request contains no breaking changes
- [X] You tested your changes locally, and they work.
- [X] There are no new Runtimes that appear.
- [X] You documented all of your changes.

## Changelog
:cl:
tweak: relocates the bos bunker entrance, moves and adds a few things around the bunker
add: re-adds the payroll safe, some warning signs etc around the bunker
/:cl:

### CODE REVIEW

File: _maps/map_files/Pahrump/Pahrump-Underground-1.dmm [hunk 1]
```diff
@@ -11838,7 +11718,14 @@
 "gKk" = (
 /obj/structure/lattice/catwalk,
 /obj/machinery/hydroponics/constructable,
-/turf/open/water,
+/turf/open/floor/plasteel/f13/vault_floor/misc/vaultrust{
+	barefootstep = "water;
```
Comment (Looreydev) [inline _maps/map_files/Pahrump/Pahrump-Underground-1.dmm:11722]:
```suggestion
	barefootstep = "water";
```

File: _maps/map_files/Pahrump/Pahrump-Underground-1.dmm [hunk 2]
```diff
@@ -11860,7 +11747,14 @@
 /obj/machinery/light/small{
 	dir = 1
 	},
-/turf/open/water,
+/turf/open/floor/plasteel/f13/vault_floor/misc/vaultrust{
+	barefootstep = "water;
```
Comment (Looreydev) [inline _maps/map_files/Pahrump/Pahrump-Underground-1.dmm:11751]:
```suggestion
	barefootstep = "water";
```

File: _maps/map_files/Pahrump/Pahrump-Underground-1.dmm [hunk 3]
```diff
@@ -12858,7 +12756,14 @@
 	brightness = 3;
 	dir = 8
 	},
-/turf/open/water,
+/turf/open/floor/plasteel/f13/vault_floor/misc/vaultrust{
+	barefootstep = "water;
```
Comment (Looreydev) [inline _maps/map_files/Pahrump/Pahrump-Underground-1.dmm:12760]:
```suggestion
	barefootstep = "water";
```

File: _maps/map_files/Pahrump/Pahrump-Underground-1.dmm [hunk 4]
```diff
@@ -13726,7 +13631,14 @@
 /obj/effect/decal/cleanable/dirt,
 /obj/machinery/hydroponics/constructable,
 /obj/structure/lattice/catwalk,
-/turf/open/water,
+/turf/open/floor/plasteel/f13/vault_floor/misc/vaultrust{
+	barefootstep = "water;
```
Comment (Looreydev) [inline _maps/map_files/Pahrump/Pahrump-Underground-1.dmm:13635]:
```suggestion
	barefootstep = "water";
```

**Review by Looreydev — CHANGES_REQUESTED** [review]

**Comment by ThyBaron** [comment]
Hello! Were the marble bust's truly necessary?

**Pull request merged**